### PR TITLE
Capping to limit or min/max allowed leaves annotation

### PR DESCRIPTION
**Pull request by schylek**

### CODE REVIEW

File: vertical-pod-autoscaler/pkg/utils/vpa/capping.go [hunk 1]
```diff
@@ -30,88 +30,127 @@ func NewCappingRecommendationProcessor() RecommendationProcessor {
 	return &cappingRecommendationProcessor{}
 }
 
+type cappingAction = string
+
+var (
+	cappedToMinAllowed cappingAction = "capped to min allowed"
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/utils/vpa/capping.go]: capped to min allowed from VPA resourcePolicy? or VPA containerPolicy
min allowed seems a bit too generic

File: vertical-pod-autoscaler/pkg/utils/vpa/capping.go [hunk 2]
```diff
@@ -30,88 +30,127 @@ func NewCappingRecommendationProcessor() RecommendationProcessor {
...
+	cappedToMaxAllowed cappingAction = "capped to max allowed"
+	cappedToLimit      cappingAction = "capped to limit"
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/utils/vpa/capping.go]: capped to request limit? Not very sure though, limit might be specific enough

**Comment by bskiba** [comment]
Do you have an example of what the annotation looks like? I wonder if we're not putting too much info in one place.

File: vertical-pod-autoscaler/pkg/utils/vpa/capping.go (hunk 1 above)
Comment (bskiba) [reply vertical-pod-autoscaler/pkg/utils/vpa/capping.go]: I was wondering if I see minAllowed I'll know immediately that this is the minAllowed from the VPA object. But I might be paranoid.

**Comment by schylek** [comment]
Annotations:    vpaUpdates=Pod resources updated by hamster-vpa:
container 0: cpu capped to max allowed, memory capped to min allowed,
cpu request, memory request

File: vertical-pod-autoscaler/pkg/utils/vpa/capping.go (hunk 2 above)
Comment (schylek) [reply vertical-pod-autoscaler/pkg/utils/vpa/capping.go]: Changed to "capped to container limit".

File: vertical-pod-autoscaler/pkg/utils/vpa/capping.go (hunk 1 above)
Comment (schylek) [reply vertical-pod-autoscaler/pkg/utils/vpa/capping.go]: ok, changed minAllowed/maxAllowed

**Pull request merged**